lib/new_relic/agent/instrumentation/action_mailbox_subscriber.rb

```diff
@@ -0,0 +1,66 @@
+# This file is distributed under New Relic's license terms.
+# See https://github.com/newrelic/newrelic-ruby-agent/blob/main/LICENSE for complete details.
+# frozen_string_literal: true
+
+require 'new_relic/agent/instrumentation/notifications_subscriber'
+
+module NewRelic
+  module Agent
+    module Instrumentation
+      class ActionMailboxSubscriber < NotificationsSubscriber
+        def start(name, id, payload)
```

Comment:
The barebones `start`, `finish`, etc. methods in this class are very encouraging. It looks like we're close to adding them to `NotificationsSubscriber` and simplifying our `Subscriber` sub-classes.

Reply:
Yep, these are mostly if not entirely identical to the ones from the ActiveStorage subscriber and pretty generic. It would be great to DRY some of these common patterns up after the conclusion of the milestone.